<launch>
	<param name="use_sim_time" value="true"/>
	<node pkg="pgslam" name="pgslam" type="pgslam" output="screen" >
		<param name="resolution" type="double" value="0.05"      />
		<param name="draw_range" type="double" value="6"         />
		<param name="map_frame"  type="string" value="map"       />
		<param name="odom_frame" type="string" value="odom"      />
		<param name="base_frame" type="string" value="base_link" />
	</node>
	<node pkg="rviz" name="rviz" type="rviz" output="screen" />
	<node pkg="rosbag" name="play" type="play" output="screen" args="$(find pgslam)/bag/mrpt_world.bag --clock" />
</launch>

